### PR TITLE
Reduce floot on startup

### PR DESCRIPTION
**Pull request by pvizeli**
## Proposed change

- Revert timeout for keep things fast as before with old backend
- Disable plugin check and core check on startup and keep it after the system is running
- Abort trust check if a backend error is happening

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:
- Link to cli pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [ ] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

File: supervisor/resolution/checks/core_trust.py [hunk 1]
```diff
@@ -32,6 +32,9 @@ async def run_check(self) -> None:
         except CodeNotaryUntrusted:
             self.sys_resolution.unhealthy = UnhealthyReason.UNTRUSTED
             self.sys_resolution.create_issue(IssueType.TRUST, ContextType.CORE)
+        except CodeNotaryBackendError:
+            _LOGGER.warning("CAS issues happening, skip tests for now")
```
Comment (ludeeus) [inline supervisor/resolution/checks/core_trust.py:36]:
```suggestion
            _LOGGER.warning("CAS backend issue, skipping check")
```

File: supervisor/resolution/checks/plugin_trust.py [hunk 2]
```diff
@@ -35,6 +35,9 @@ async def run_check(self) -> None:
                 self.sys_resolution.create_issue(
                     IssueType.TRUST, ContextType.PLUGIN, reference=plugin.slug
                 )
+            except CodeNotaryBackendError:
+                _LOGGER.warning("CAS issues happening, skip tests for now")
```
Comment (ludeeus) [inline supervisor/resolution/checks/plugin_trust.py:39]:
```suggestion
                _LOGGER.warning("CAS backend issue, skipping check")
```

File: supervisor/resolution/checks/core_trust.py [hunk 3]
```diff
@@ -32,6 +32,9 @@ async def run_check(self) -> None:
         except CodeNotaryUntrusted:
             self.sys_resolution.unhealthy = UnhealthyReason.UNTRUSTED
             self.sys_resolution.create_issue(IssueType.TRUST, ContextType.CORE)
+        except CodeNotaryBackendError:
+            _LOGGER.warning("CAS backend issue, skipping check")
+            return
```
Comment (ludeeus) [inline supervisor/resolution/checks/core_trust.py:37]:
```suggestion
```
There is nothing more after this, so no need to return here

File: supervisor/resolution/checks/plugin_trust.py [hunk 4]
```diff
@@ -35,6 +35,9 @@ async def run_check(self) -> None:
                 self.sys_resolution.create_issue(
                     IssueType.TRUST, ContextType.PLUGIN, reference=plugin.slug
                 )
+            except CodeNotaryBackendError:
+                _LOGGER.warning("CAS backend issue, skipping check")
+                return
```
Comment (ludeeus) [inline supervisor/resolution/checks/plugin_trust.py:40]:
```suggestion
```
There is nothing more after this, so no need to return here

**Review by ludeeus — APPROVED** [review]

File: supervisor/resolution/checks/plugin_trust.py (hunk 4 above)
Comment (pvizeli) [reply supervisor/resolution/checks/plugin_trust.py:40]: it's on a loop

Comment (ludeeus) [reply supervisor/resolution/checks/plugin_trust.py:40]: Ah, right 👍

**Pull request merged**